Lib/ufo2ft/featureWriters/kernFeatureWriter2.py

```diff
@@ -528,14 +546,16 @@ def _registerKernLookups(self, feature, lookups):
                     )
         elif ltrLookups:
             if not (rtlScripts or distScripts):
-                ast.addLookupReferences(feature, ltrLookups)
+                if ltrLookupsRest:
+                    ast.addLookupReferences(feature, ltrLookupsRest)
```

Comment:
are you sure these lines can be hit? my understanding is that the condition for hitting this lines is that: 1) there are only some ltrLookps, no rtlLookups; 2) there are no rtlScripts nor distScripts defined and 3) there is some `ltrLookupRest` still available that haven't already been registered globally as part of the earlier dfltLookups (i.e. `ltrLookupRest = [lkp for lkp in ltrLookups if lkp not in dfltLookups]`).

Do the test hit this condition? Is it even hittable?

Reply:
If I replace `addLookupReferences` by `assert None`, many tests fail. The `dfltLookups` are only emitted when `"DFLT" in lookups`, and various tests don't trigger that path.

Reply:
note that the doubly referenced lookups that you're trying to avoid here should be fixed (ignored) in latest fonttools
https://github.com/fonttools/fonttools/pull/3520